paper-progress.html

```diff
@@ -70,10 +70,11 @@
 
 Custom property                                  | Description                                 | Default
 -------------------------------------------------|---------------------------------------------|--------------
-`--paper-progress-container-color`               | Mixin applied to container                  | `--google-grey-300`
```

Comment:
It's actually still there, only a couple of lines lower, next to the other color properties.